plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -448,7 +453,9 @@ async function setupSonataflowService(
   logger: Logger,
   jiraConfig?: JiraConfig,
 ) {
-  const sonataFlowResourcesAbsPath = resolve(`${sonataFlowResourcesPath}`);
+  const sonataFlowResourcesAbsPath = resolve(
+    join(sonataFlowResourcesPath, default_workflows_path),
```

Comment:
With this PR, we can remove the `plugins/orchestrator-backend/workflows` folder, right?

Reply:
yes, before merging I'll do that